test/pytest/test_data/streaming/model_config.yaml

```diff
@@ -0,0 +1,2 @@
+continuousBatching: true
+modelId: gpt2
```

Comment:
it's better to add handler section. eg.

handler:
    modelId: gpt2

Reply:
done